### PR TITLE
Typed tests

### LINKED ISSUE #54: Cache Control behavior when set to null
Test + add to tests what happens when `bypassCache`, `browserTTL`, or/and `edgeTTL` or set to null

Document the behavior in the readme

### PR DESCRIPTION
**Pull request by victoriabernard92**
Fixes #54 

Moved tests to typescript and document/test what happens when `edgeTTL` is null

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,9 +6,9 @@
   "types": "./dist/index.d.ts",
   "scripts": {
     "build": "tsc -d",
-    "build:watch": "npm run build -w",
+    "build:watch": "yarn build -w",
```
Comment (victoriabernard92) [inline package.json]: watch requires yarn. devs have option to just build if they don't want to use watch

Comment (EverlastingBugstopper) [reply package.json]: I'm generally opposed to having `npm` and `yarn` co-exist in the same directory so if we require yarn we should remove package-lock and replace it with yarn.lock. 

Additionally, you can just run `yarn build -w` and the `-w` will be passed to the `tsc -d` command and will watch so it might be unnecessary to have the `yarn` command in the package.json itself.

File: src/test/getAssetFromKV.ts [hunk 2]
```diff
@@ -185,3 +180,20 @@ test('getAssetFromKV no result throws an error', async t => {
   const event = getEvent(new Request('https://blah.com/random'))
   await t.throwsAsync(getAssetFromKV(event))
 })
+test('getAssetFromKV browser cache set to nul', async t => {
```
Comment (EverlastingBugstopper) [inline src/test/getAssetFromKV.ts]: `nul`->`null`

Comment (victoriabernard92) [reply src/test/getAssetFromKV.ts]:
```suggestion
test('getAssetFromKV browser cache set to null', async t => {
```
